ci/unit/test_post_always.sh

```diff
@@ -6,7 +6,7 @@
 set -uex
 
 if [ -e ./.build_vars.sh ]; then
-  # shellcheck disable=SC1091
+  # shellcheck source=../../.build_vars.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]